client/components/checklist/style.scss

```diff
@@ -2,20 +2,38 @@
 	display: flex;
 	flex-direction: row;
 
+	@include breakpoint( '<480px' ) {
+		flex-direction: column;
+		justify-content: flex-end;
+	}
+
+	&.card {
+		padding: 0;
+	}
+
 	&-main {
 		display: flex;
 		flex: 1 1;
 		flex-direction: column;
 		align-items: stretch;
+		padding: 16px 24px;
+		@include breakpoint( '<480px' ) {
+			padding: 16px;
```

Comment:
Where possible it's better to target mobile styles in the base rules, and then modify for larger screens, so that mobile clients don't have to do so much computation of styles.

Reply:
I toootally agree and that's how I typically do.

In this case, I opted to do another way around to stay consistent with how the file has been structured until now. Breakpoints are hard to grok and mixture of up/down breakpoints is even harder.

We have a visual refactor coming up for these components and I'd like to sort this out at that point in time for all the breakpoints at once.

Thanks!